
Do not print backup boot sector diff in non interactive mode to avoid log spam.

--- a/src/boot.c	2017-01-23 02:16:58.000000000 +0100
+++ a/src/boot.c	2020-02-05 18:32:16.000000000 +0100
@@ -174,6 +174,9 @@ static void check_backup_boot(DOS_FS * f
 	char buf[20];
 
 	printf("There are differences between boot sector and its backup.\n");
+	if (!interactive)
+	    printf("This is mostly harmless.\n");
+	else {
 	printf("This is mostly harmless. Differences: (offset:original/backup)\n  ");
 	pos = 2;
 	for (p = (uint8_t *) b, q = (uint8_t *) & b2, i = 0; i < sizeof(b2);
@@ -188,6 +191,7 @@ static void check_backup_boot(DOS_FS * f
 		first = 0;
 	    }
 	}
+	}
 	printf("\n");
 
 	if (interactive)
